RELEASES_CHECKLIST.md

```diff
@@ -69,6 +69,7 @@ in the future.
         is not necessary.
     - We don't want `cargo-release` to create any releases or push any code, we'll do
       that manually once we've actually published to `crates.io`.
+    - Note that `ink_sandbox` is released separetly and is excluded via the `--exclude` flag.
```

Comment:
We could still include ink_sandbox in the ink release (sometimes even it will be necessary because e2e depends on it). The difference is that it can be released on its own independent of the other crates, and therefore has its own version number.

*This is a repost of a comment I added on the commit, but it doesn't show on this PR*

Reply:
Thks Andrew, I have updated the checklist changes, with your comment